lib/handlers/authenticate-handler.js

```diff
@@ -216,8 +216,8 @@ AuthenticateHandler.prototype.getAccessToken = function(token) {
  */
 
 AuthenticateHandler.prototype.validateAccessToken = function(accessToken) {
-  if (accessToken.accessTokenExpiresAt && !(accessToken.accessTokenExpiresAt instanceof Date)) {
-    throw new ServerError('Server error: `expires` must be a Date instance');
+  if (!accessToken.accessTokenExpiresAt && !(accessToken.accessTokenExpiresAt instanceof Date)) {
```

Comment:
The `&&` should be changed to `||`, right?

We want the check to fail if either `accessTokenExpiresAt` isn't set ***or*** it's not an instance of `Date`.

Actually, you can remove the first part altogether since `instanceof` returns `false` if `accessTokenExpiresAt` is `undefined`/`null`/whatever.

Also, the first part of the check three lines down is now superfluous since `accessTokenExpiresAt` is always set.